Disable new profiles option (#971)
zowe/zowe-native-proto: packages/cli/CHANGELOG.md

Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.

## Recent Changes

> - Added `disableCreateNewProfile` option to `promptForProfile` to restrict profile creation during setup operations and allow selection from existing profiles only. [#971](https://github.com/zowe/zowex/pull/971)

Reviewer: ```suggestion
- Added `disableCreateNewProfile` option to the `promptForProfile` function to restrict profile creation during set-up operations. This allows selection from existing profiles only. [#971](https://github.com/zowe/zowex/pull/971)
```
